### DISCUSSION TITLE
Why is writing to the SSD still blocked？

### DISCUSSION THREAD

**Discussion by lrlbh** [Chia-Network/bladebit#68] [Q&A]
- 1T  3000M/S SSD.

- The same goes for trying to combine two 3000M/S SSDS into raid0.  

- This is a command
    bladebit  -n 9999 -t 48 -f XXX -c XXX  ~/nvme0/

- This is the scene
    Waiting for last plot to finish being written to disk...
    Previous plot /home/XXXX/nvme0/XXXX.plot.tmp finished writing to disk:

    Finished plotting in 1107.27 seconds (18.45 minutes).
    Finished plotting in 1180.66 seconds (19.68 minutes).
    Finished plotting in 1177.99 seconds (19.63 minutes).
    Finished plotting in 1186.84 seconds (19.78 minutes).
    Finished plotting in 1179.29 seconds (19.65 minutes).
    Finished plotting in 1175.77 seconds (19.60 minutes).
    Finished plotting in 1174.41 seconds (19.57 minutes).
    Finished plotting in 1182.58 seconds (19.71 minutes).
    Finished plotting in 1175.75 seconds (19.60 minutes).
    Finished plotting in 1192.12 seconds (19.87 minutes).

**Comment by xhamma** [comment]
I'm not really sure I understand the problem... but you said that you have a 1TB drive and there are 10 plots shown to have been completed. at ~100GB a piece, that would fill a 1TB drive, so I would expect it to be waiting for more space to continue at that point. I'm not sure why you would specify 9999 plots if this is the case. Do you have a script to move them off onto your farming drives after they are created?
